### PR TITLE
coding style: update

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: CODING_STYLE.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 LXC Coding Style Guide
 ======================
 
-In general the LXC project follows the Linux kernel coding style.  There are
-however are a few differences, these are outlined in this document.
+In general the LXC project follows the Linux kernel coding style.  Hwoever,
```
Comment (tomponline) [inline CODING_STYLE.md]: @brauner typo

**Comment by tomponline** [comment]
@brauner makes sense thanks

File: CODING_STYLE.md (hunk 1 above)
Comment (brauner) [reply CODING_STYLE.md]: No, that's how this word is written. For sure. :D

Comment (brauner) [reply CODING_STYLE.md]: fixed

**Pull request merged**